src/services/articleHotServices.ts

```diff
@@ -0,0 +1,21 @@
+import prisma from '@/db';
```

Comment:
[Required] 既有的 service 用 db 當作命名，建議保持一致
```suggestion
import db from '@/db';
```